hack/update-generated-bindata.sh

```diff
@@ -32,7 +32,8 @@ pushd "${OS_ROOT}" > /dev/null
     examples/heapster/... \
     examples/prometheus/... \
     examples/service-catalog/... \
-    pkg/image/admission/imagepolicy/api/v1/...
+    pkg/image/admission/imagepolicy/api/v1/... \
+    test/testdata/templateservicebroker/...
```

Comment:
hack/update-generated-bindata.sh already pulls from many locations in examples for extended tests, a new one would need to be added, that's all